fix repo directory for apt (#44323)
elastic/beats: docs/reference/filebeat/setup-repositories.md

    sudo apt-get install apt-transport-https
    ```

> 3. Save the repository definition to _/etc/apt/sources.list.d/elastic-9.x.list_:

Reviewer: ```suggestion
3. Save the repository definition to _/etc/apt/sources.list.d/elastic-{{major-release}}.list_:
```

    ```shell subs=true

>     echo "deb https://artifacts.elastic.co/packages/9.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-9.x.list

Reviewer: ```suggestion
    echo "deb https://artifacts.elastic.co/packages/{{major-release}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-release}}.list
```

    ```

    :::{note}
    The package is free to use under the Elastic license. An alternative package which contains only features that are available under the Apache 2.0 license is also available. To install it, use the following sources list:

    ```shell subs=true

>     echo "deb https://artifacts.elastic.co/packages/oss-9.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-9.x.list

Reviewer: ```suggestion
    echo "deb https://artifacts.elastic.co/packages/oss-{{major-release}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-release}}.list
```
